### ISSUE TITLE
Getting Threats Detected

### ISSUE THREAD

**Issue by Zulandratronics** [Aioros/console-streaming-server#6]
It took a while to show up. It's trojan.script/wacatac.B!ml

**Comment by Aioros** [comment]
Yeah, I see that too. Apparently it's a common issue with Windows Defender getting false positives on downloaded archives. I'm trying to see what I can do about it. I'll post updates here, thanks.

**Comment by Aioros** [comment]
No solution found yet. I tried updating some of the dependencies, repackaging in a few different ways, including a new release (v1.2.8), and I still see the issue. Frustratingly, some test releases during the process did not trigger Defender, but I am unable to replicate it.

I submitted the latest zip to Microsoft as a false positive, and hopefully that might solve the issue for that file relatively soon, but 1) the only thing I can suggest in the meantime is "disable Windows Defender", which sucks, and 2) is this going to happen randomly on each new release? No idea at the moment.

**Comment by Zulandratronics** [comment]
No worries. I just wish I could help somehow.

**Comment by Aioros** [comment]
Let me at least give a reasonable workaround in the meantime: you can add a folder exclusion in Windows Defender (either a folder you already have, or a new one that you would create for this), and anything you put in there won't be touched. The setting in Windows Defender is under "Virus & thread protection" -> "Virus & thread protection settings" -> "Exclusions" -> "Add or remove exclusions".
Then you only have to tell your browser to download the zip in that excluded folder and it should be ok.
